preflight-checks/src/utils/tracks/analytics/clients/tracks.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import fetch from 'node-fetch';
```

Comment:
probably a good candidate for `vip-go-node-internal` once we figure it out as a package